packages/reactive-element/src/reactive-element.ts

```diff
@@ -668,12 +674,7 @@ export abstract class ReactiveElement
     // is called before `finalize`, we ensure finalization has been kicked off.
     this.finalize();
     this.elementProperties.set(name, options);
-    // Do not generate an accessor if the prototype already has one, since
-    // it would be lost otherwise and that would never be the user's intention;
-    // Instead, we expect users to call `requestUpdate` themselves from
-    // user-defined accessors. Note that if the super has an accessor we will
-    // still overwrite it
-    if (!options.noAccessor && !this.prototype.hasOwnProperty(name)) {
+    if (!options.noAccessor) {
```

Comment:
For my own understanding, this is where the wrapping is happening. `!this.prototype.hasOwnProperty(name)` would previously only wrap class fields and skip accessors or user defined get/set.
Now explicit getters/setters and `accessor` keyword cases pass through this code path (in addition to the class fields).